### ISSUE TITLE
Error Cannot read property 'quasar' of undefined while use with Typescript

### ISSUE THREAD

**Issue by tindecken** [quasarframework/vue-cli-plugin-quasar#19]
**Describe the bug**
Cannot read property 'quasar' of undefined while use with Typescript
**To Reproduce**
1. Use Vue CLI to create new project with language Typescript
2. Add this plugin
3. Run `yarn serve`

![image](https://user-images.githubusercontent.com/4727197/63225410-da009700-c1f9-11e9-91bf-e1629280d551.png)

**Expected behavior**
No error occur

**Actual Result**
Console error as below
```
D:\Develop\tcm (master -> origin)
λ yarn serve
yarn run v1.17.3
$ vue-cli-service serve
 ERROR  TypeError: Cannot read property 'quasar' of undefined
TypeError: Cannot read property 'quasar' of undefined
    at module.exports (D:\Develop\tcm\node_modules\vue-cli-plugin-quasar\index.js:2:29)
    at plugins.forEach (D:\Develop\tcm\node_modules\@vue\cli-service\lib\Service.js:80:7)
    at Array.forEach (<anonymous>)
    at Service.init (D:\Develop\tcm\node_modules\@vue\cli-service\lib\Service.js:79:18)
    at Service.run (D:\Develop\tcm\node_modules\@vue\cli-service\lib\Service.js:206:10)
    at Object.<anonymous> (D:\Develop\tcm\node_modules\@vue\cli-service\bin\vue-cli-service.js:36:9)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this **command.**
```
**Platform (please complete the following information):**
OS: Windows
Node: 10.15

**Comment by Laegel** [comment]
I had the same issue (pretty much the same stack, though I'm unsure it's the real problem here), but I managed to solve it one hour ago.
At first, I've been using the Vue UI to install my app and the plugins, but when I had a look to this repo README, I saw that part:

> ```
> cd my-app
> vue add quasar
> ```
> It will ask you if you want the plugin to replace some existing files. It is recommended that you do it if you wish to have an example so you can quickly develop your app.

In fact, when using the Vue UI, **it did not ask to replace some existing files**.

So, @tindecken , you may be able to fix your issue by executing the `vue add quasar` command.

But I think there's definitely an issue with the plugin used within the UI.

**Comment by rstoenescu** [comment]
Fix will be available today in "vue-cli-plugin-quasar" v3.0.0.
Please read the upgrade guide (after released).
